agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/messaging/MessageEncoder.java

```diff
@@ -299,6 +299,7 @@ default EncodedMessage encode(
 
   /**
    * Get the service to encode messages.
+   *
```

Comment:
Why? Is this your formatter?

Reply:
No, it's the mvn formatter and this is the correct formatting.
It used to be wrong and was fixed in #86 .
As this branch origins from an older version of develop, it's shown as a change.
Actually, it's the same as in the current develop branch.